requirements/requirements.dev.txt

```diff
@@ -1,8 +1,8 @@
-setuptools==40.2.0
-wheel==0.31.1
-pylint==2.1.1
-typing==3.6.6
-mypy==0.620
-sphinx==1.7.6
-pillow==5.2.0
-pytest==3.7.3
+setuptools>=40.2.0
+wheel>=0.31.1
+pylint>=2.1.1
+ipython>=6.2.1
```

Comment:
Why did you remove explicit dependency on Typing module? It's used in the code and it's unclear from dependency on iPython that it's used. Moreover, I'm concerned why do we have iPython here - is it required for build ?